src/smart-app-banner.js

```diff
@@ -0,0 +1,181 @@
+var extend = require('xtend/mutable');
```

Comment:
What was the reason to place files into `./src` folder, except for a habit to do so?